packages/create-svelte/shared/+vitest+default/src/lib/Counter.test.ts

```diff
@@ -0,0 +1,31 @@
+import { render, screen, fireEvent } from '@testing-library/svelte';
```

Comment:
This perhaps deserves some discussion with the other maintainers, but my belief is we probably would not want to add a dependency on `@testing-library/svelte`. I've seen a few different solutions used for component testing and it's not obvious to me that there's a clear leader. For now, I would just have Vitest test a simple function and not test any `.svelte` file until there's a clearer consensus around how'd we like to recommend testing individual components.

I started a discussion about this here: https://github.com/sveltejs/kit/discussions/5285